packages/react-query/package.json

```diff
@@ -27,7 +27,7 @@
     "esm"
   ],
   "scripts": {
-    "build": "rm -rf dist esm && tsc -p tsconfig.build.json --declaration --emitDeclarationOnly --declarationDir dist && rollup -c rollup.config.js",
+    "build": "tsup src/index.ts --format cjs,esm --dts --minify",
```

Comment:
we have to support cjs!?

For a better `node` ecosystem, I want to support only `esm`, but I'm also worried that certain people might prefer `cjs`

Reply:
I think we need to support cjs yet. because there is creact-react-app, jest(most popular depenency for every node developer are using) that support only cjs officially, not [esm that is experimental supported yet](https://jestjs.io/docs/ecmascript-modules) yet.

for example, swiper(support only esm) have thes issues
1. https://swiperjs.com/react#usage-with-create-react-app
2. https://github.com/nolimits4web/swiper/issues/4871
3. https://github.com/nolimits4web/swiper/discussions/4969

I want to promote library users use esm too, but at the same time, I don't want to make library users go away from us by forcing them use esm

Reply:
I'm fully agreed with your opinion😄